Sources/Operators.swift

```diff
@@ -241,7 +241,7 @@ public func <~~ (key: String, json: JSON) -> [URL]? {
 /**
 Encode custom operator.
 */
-infix operator ~~> { associativity left precedence 150 }
+infix operator ~~> : MultiplicationPrecedence
```

Comment:
[SE-0077](https://github.com/apple/swift-evolution/blob/master/proposals/0077-operator-precedence.md) discuss this change, it has the list of operator groups.

I picked that one because it shouldn't change the current behavior.

Checking a [list of swift operators precedence](http://nshipster.com/swift-operators/) I saw the one that behaves like ~~> (same associativity and precedence) are the ones in the multiplication group.

Maybe we should define our own group?

``` swift
precedencegroup DecodingPrecedence {
  associativity: left
  higherThan: CastingPrecedence
}
```